students/jvirtue/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,13 @@
+# Print fizz for multiples of 3 and Buzz for multiples of 5
+for x in range (100):
+    if x == 0:
+        print(0)
+    elif x%3 == 0 and x%5 == 0:
+        print("FizzBuz")
+    elif x%3 == 0:
+        print("Fizz")
+    elif x%5 == 0:
+        print("Buzz")
+    else:
+        print(x)
```

Comment:
Looks good - but only goes to 99.  Should go to 100.

Reply:
Made updates to this pull request https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/1347